Assets/Plugins/Source/Core/EOSManager.cs

```diff
@@ -675,12 +680,10 @@ public void Init(IEOSCoroutineOwner coroutineOwner, string configFileName)
 
                 InitializeOverlay(coroutineOwner);
 
-                // Default back to quiet logs
-#if UNITY_EDITOR
-                SetLogLevel(LogCategory.AllCategories, LogLevel.VeryVerbose);
-#else
-                SetLogLevel(LogCategory.AllCategories, LogLevel.Warning);
-#endif
+                for (int logCategoryIndex = 0; logCategoryIndex < logLevelList.Count; logCategoryIndex++)
+                {
+                    SetLogLevel((LogCategory)logCategoryIndex, logLevelList[logCategoryIndex]);
+                }
```

Comment:
Same question here as above; it does not seem like between each of these loops the log level is changing, so I'm confused about why each needs to be run.

Reply:
These are for the platforms that uses dynamic binding, which are excluded in the earlier call.
Those using dynamic binding can not use SetLogLevel before the LogInterface gets initialized, or else it would crash.